src/main/webapp/i18n/de/student-dashboard.json

```diff
@@ -177,7 +179,20 @@
                 "graded": "Bewertet",
                 "notGraded": "Nicht bewertet",
                 "rated": "Bewertet",
-                "practice": "Übung"
+                "practice": "Übung",
+                "assessmentType": "Bewertungsart:",
+                "submissionDue": "Abgabe bis:",
+                "submissionDueTooltip": "Die Abgabe ist bis {{date}} möglich.",
+                "testsAfterDueDate": "Testdurchlauf nach Abgabefrist:",
+                "testsAfterDueDateTooltip": "Die Tests werden an {{date}} erneut durchlaufen.",
+                "exampleSolutionPublicationDay": "Veröffentlichung der Beispiellösung:",
+                "exampleSolutionPublicationDayTooltip": "Die Beispiellösung wird an {{date}} veröffentlicht.",
```

Comment:
```suggestion
                "exampleSolutionPublicationDayTooltip": "Die Beispiellösung wird am {{date}} veröffentlicht.",
```

Reply:
It's not only the date but also the time that is inserted here. So "am" doesn't really fit, does it?
What do you think about my solution?

Reply:
I guess `am` fits more than `an` :D

Reply:
e.g. "Die Beispiellösung wird am 15.12 14:00 veröffentlicht"

Reply:
The ideal solution would be to have "wird am 15.12 um 14:00 Uhr" veröffentlicht, but I am not sure if this is easily doable.
But I also agree that "am" sounds better here than "an".